### PR TITLE
create get announcements

### PR DESCRIPTION
**Pull request by Midnite1212**
Finished get all announcements, by id, custom host and routes.

### CODE REVIEW

File: server/api/controllers/sermons/get-all-announcements.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+const { DateTime } = require('luxon');
```
Comment (albertparedandan) [inline server/api/controllers/sermons/get-all-announcements.js:1]: small note, PR should merge to release not main branch. I changed yours already so it's fine

File: server/api/controllers/sermons/get-all-announcements.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+const he = require('he');
+
+const transformAnnouncement = (announce) => {
```
Comment (albertparedandan) [inline server/api/controllers/sermons/get-all-announcements.js:4]: it'd be good to look into some of the ways we're using caching and helpers to do a lot of this. can talk to you more about it thru call and help walk through it with you

File: server/api/controllers/sermons/get-all-announcements.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  return {
+    id: announce.id,
+    slug: announce.slug,
+    title : he.decode(announce.title.rendered),
+    startDate: DateTime.fromFormat(announce.date, "LLLL MM dd yyyy"),
```
Comment (albertparedandan) [inline server/api/controllers/sermons/get-all-announcements.js:9]: any reason why the formats for the time are different? ideally it should just be `2021-07-04T21:24:03.000+08:00` as an example

File: server/api/controllers/sermons/get-all-announcements.js (hunk 2 above)
Comment (albertparedandan) [reply server/api/controllers/sermons/get-all-announcements.js:4]: also maybe `annoucement` instead of `announce`

**Review by albertparedandan — CHANGES_REQUESTED** [review]

File: server/api/controllers/sermons/get-all-announcements.js (hunk 1 above)
Comment (Midnite1212) [reply server/api/controllers/sermons/get-all-announcements.js:1]: okk thanks

File: server/api/controllers/sermons/get-all-announcements.js (hunk 3 above)
Comment (Midnite1212) [reply server/api/controllers/sermons/get-all-announcements.js:9]: that's the one im not rly sure, cuz the json file format was in unicode like "Fri \u00b7 7\/30\/2021" not sure whats the format for this...

**Review by tyleung — DISMISSED** [review]

**Review by tyleung — COMMENTED** [review]
use https in custom.js

**Review by tyleung — APPROVED** [review]

**Review by albertparedandan — APPROVED** [review]

**Pull request merged**